Java/Fundamentals/OOPs Principals/OOPs_Principals_documentation.md

```diff
@@ -0,0 +1,88 @@
+# OOP
+------------
+## Contents
+1. Introduction to class and object 
+2. What is a class ?
+3. What is an object?
+4. Difference between Classes and objects in Java
+5. Memory allocation 
+ 
+-------------
+## Introduction to class and object 
+- Classes and objects are one of the most fundametal things of many Java application and frameworks. 
```

Comment:
grammatical error

change it to `fundamental units`

Reply:
okay i will change it . Is there anything else